src/output_parser/Oyente.py

```diff
@@ -32,7 +32,9 @@ def parse(self, str_output):
             elif "INFO:symExec:	  " in line:
                 (key, value) = self.extract_result_line(line)
                 current_contract[key] = value
-            elif current_contract is not None and "INFO:symExec:" + current_contract['file'] in line:
+            elif current_contract is not None and current_contract['file'] in line:
+                if "INFO:symExec:" not in line:
```

Comment:
can this be merged with the previous if? That is:

```
not current_contract and current_contract['file'] in line and "INFO:symExec:" not in line
```

Reply:
No, if this is true then you will be missing the first vuln, this one:
```
INFO:symExec:/dataset/reentrancy/0x01f8c4e3fa3edeb29e514cba738d87ce8c091d3f.sol:57:17: Warning: Re-Entrancy Vulnerability.
                Log.AddMessage(msg.sender,_am,"Collect")
```

Reply:
OK. That's what I thought. wouldn't it be better to simplify this check using a regular expression (re2)?

Reply:
Maybe, I dont use that

Reply:
No worries. :)